Fix dark mode issue of helpful Search and search results in light mode

subhas-pramanik-09 opened the pull request:
Previously #4333  In this PR, the dark mode in search, helpful search, search results implemented. But after "Generalised Dark Mode into General Theme #4348" this PR MB has moved "darkmode.css" to "themes.css". 

So that, in light mode, helpful search and search results looks like dark mode.

Before : 
![Screenshot 2025-02-09 190838](https://github.com/user-attachments/assets/7541cefc-9bf1-4c60-a2b2-1a7f7b5c51ef)
![Screenshot 2025-02-09 190849](https://github.com/user-attachments/assets/10293c6c-d2fb-4af5-8900-dbebb418ef9c)

After this change properly dark mode and light mode has applied. 

Now : 
![Screenshot 2025-02-09 194516](https://github.com/user-attachments/assets/580ae153-6750-4698-93b7-fd5eb0011edc)
![Screenshot 2025-02-09 194528](https://github.com/user-attachments/assets/158c6c89-9a83-4c39-a13f-c7081701d2c8)

subhas-pramanik-09 commented:
@walterbender Sir, please review

arjunjayan999 commented:
Hey @subhas-pramanik-09 thanks. I thought I put it back after encountering the issue but maybe I forgot, maybe keep the not(.dark) CSS in activities.css as it was before to maintain consistency. activities.css contains the light mode styling, themes.css currently has the dark mode styling.

subhas-pramanik-09 commented:
> Hey @subhas-pramanik-09 thanks. I removed it while refactoring the code but forgot to put it back. Maybe keep the not(.dark) CSS in activities.css as it was before to maintain consistency? activities.css contains the light mode styling, themes.css currently has the dark mode styling.

Okay, as activities.css contains by default styling of light mode so I moved this part to activities.css

arjunjayan999 commented:
yep thanks again, sorry for removing your PR, not intentional.

subhas-pramanik-09 commented:
> yep thanks again, sorry for removing your PR, not intentional.

Welcome brother, everything is okay only this part was not working
